packages/web/src/components/lineup/CardLineup.tsx

```diff
@@ -37,7 +37,7 @@ const DesktopCardContainer = ({
             <Draggable
               key={`draggable-${card.props.playlistId}`}
               text={card.props.primaryText}
-              kind={card.props.isAlbum ? 'album' : 'playlist'}
+              kind={card.props.isPlaylist ? 'playlist' : 'album'}
```

Comment:
This is the real fix right here - the desktop CollectionsPage component was passing in the `isPlaylist` prop but this component was looking for an `isAlbum` prop https://github.com/AudiusProject/audius-client/blob/d8cb1f704d43647389ddb58664ea743b59e23bd9/packages/web/src/pages/explore-page/components/desktop/CollectionsPage.tsx#L88-L89

I updated the mobile `Card` and `CollectionsPage` components to have the `isPlaylist` prop as well so that they match the desktop ones